server/middlewares/frontendMiddleware.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const path = require('path');
+
+// Dev middleware
+const addDevMiddlewares = (app, options) => {
+  const webpackDevMiddleware = require('webpack-dev-middleware');
+  const webpackHotMiddleware = require('webpack-hot-middleware');
```

Comment:
`npm install --production` to skip dev dependencies in prod (e.g. after deploying).